src/applications/IMUCalibDumper/app_IMUCalibDumper.xml

```diff
@@ -0,0 +1,32 @@
+<application>
+    <name>IMUCalibDumper</name>
+    <description>Dump IMU data for calibration</description>
+    <version>1.0</version>
+
+<!--MODULES-->
+<module>
+    <name>vPreProcess</name>
+    <parameters> --width 304 --height 240 --split_stereo true --undistort false --filter_spatial true --filter_temporal true</parameters>
```

Comment:
these parameters for vPreProcess aren't needed and could cause crashes in some cases. I would suggest having no parameters here should be the functionality you want